src/MetadataStorage.ts

```diff
@@ -26,11 +27,21 @@ export interface RepositoryMetadata {
   entity: Function;
 }
 
+export interface Config {
+  validate?: boolean;
```

Comment:
Again, I think `validate` is not specific enough, `runValidations` or `runModelValidations` would be better.

Reply:
`validateOnSave`?

Reply:
`validateModels`? I don't think `onSave` is specific enough in this case. Why not `runModelValidations`?

Reply:
I think `validateModels` will work 😄 `runModelValidations` feels a little verbose but that might just be me,

Quick question. Is model and collection used interchangeably? Should it be `validateModels` or `validateCollections`?

Reply:
A "Model" is a blueprint of the object that will be stored in firestore (AKA the Class). A "Collection" === "Firestore Collection" is just the model with the Collection attribute.

Why they're not the same? You might have models for Subcollections/Relationships that might not be represented as Collections in Firestore.

Reply:
Okay that makes sense, thanks for clarifying 😄